webpack.config.js

```diff
@@ -1,69 +1,54 @@
 const path = require("path");
 const webpack = require("webpack");
-const fableUtils = require("fable-utils");
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
-
-function resolve(filePath) {
-    return path.join(__dirname, filePath)
-}
-
-var babelOptions = fableUtils.resolveBabelOptions({
-    presets: [
-        ["env", {
-            "targets": {
-                "browsers": ["last 2 versions"]
-            },
-            "modules": false
-        }]
-    ]
-});
-
-var isProduction = process.argv.indexOf("-p") >= 0;
-console.log("Bundling for " + (isProduction ? "production" : "development") + "...");
-
 var commonPlugins = [
     new HtmlWebpackPlugin({
-        filename: resolve('./output/index.html'),
-        template: resolve('./src/index.html')
+        filename: 'index.html',
+        template: './src/index.html'
     })
 ];
 
-module.exports = {
-    devtool: undefined,
-    entry: isProduction ? // We don't use the same entry for dev and production, to make HMR over style quicker for dev env
+module.exports = function (evn, argv) {
+  var isProduction = argv.mode === "production";
```

Comment:
I would handle the case where `mode` isn't define. So by default we set the mode to `development`.

Like that we can run `dotnet fable webpack-dev-server` if we want to run it by hand. Which we should not do because we have Fake here.